From f75e866bf79a56524b159abd76b8abca21c438d4 Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham.Lam" <loc.lam.px@renesas.com>
Date: Fri, 23 Aug 2019 13:42:22 +0700
Subject: [PATCH 290/405] arm64: dts: renesas: r8a774b1-hihope-rzg2n-ex: Enable
 PCIE1 as default instead of SATA

In RZ/G2N, we can choose the output between PCIE1 and SATA by
GP7_03 and SW43=OFF with following selections:

GPIO7_03 = HIGH -> PCIE1
GPIO7_03 = LOW -> SATA
Currently, we enable PCIE1 as default environment.

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts b/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
index ccab6bc..1e8185e 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
@@ -8,6 +8,7 @@
 
 #include "r8a774b1-hihope-rzg2n.dts"
 #include "hihope-rzg2-ex.dtsi"
+#include <dt-bindings/gpio/gpio.h>
 
 / {
 	model = "Hoperun Technology HiHope RZ/G2N extension board \
@@ -23,3 +24,12 @@
 &pciec1 {
 	status = "okay";
 };
+
+&gpio7 {
+	pcie-en-gpio {
+		gpio-hog;
+		gpios = <3 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "pcie-en-gpio";
+	};
+};
-- 
2.7.4

